### ISSUE TITLE
[fast-xml-parser]: package dependency issue 

### ISSUE THREAD

**Issue by meetamaity29** [sst/sst#3265]
---
**@aws-sdk/client-sts** package uses fast-xml-parser version 4.1.2 on it's dependencies. Which is causing high vulnerability issue. I believe **fast-xml-parser** package has a fix for it on 4.2.5 version. SST needs to be updated with this package.

**Here is the Discord Issue link :** https://discord.com/channels/983865673656705025/1145967640595148803

**Comment by sshadmand** [comment]
Thanks for originally posting this in Aug 23 @meetamaity29 

To be more specific this is at level high

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ fast-xml-parser vulnerable to ReDOS at currency parsing      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ fast-xml-parser                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.4.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ sst                                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ sst > @aws-sdk/client-lambda > @aws-sdk/client-sts >         │
│               │ fast-xml-parser                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1098559                     │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble
